bin/generate.dart

```diff
@@ -73,6 +73,14 @@ ArgParser _generateArgParser(GenerateOptions? generateOptions) {
       help: 'Support json or keys formats',
       allowed: ['json', 'keys']);
 
+  parser.addFlag(
+    'skip-object-keys',
+    abbr: 'x',
+    defaultsTo: false,
```

Comment:
default true, its keys newer use, maybe ))

Reply:
(sorry for delay and thanks for review). I've wanted to be by default false as it is "new feature" and would "break" current behavior..

Reply:
ok